core/core-funcs.el

```diff
@@ -140,14 +140,29 @@ Supported properties:
        `((dolist (val ',def-key)
           (define-key (eval (car val)) (kbd (cdr val)) ',func)))))))
 
+(defun spacemacs/prettify-org-buffer ()
```

Comment:
Thanks for extracting this function, I just had a chance to try it out.  It doesn't seem to include everything that I was thinking though, like the `*` -> `bold`, `~` -> `org-kbd`, etc.  This stuff:
```elisp
      (setcar (nthcdr 2 org-emphasis-regexp-components) " \t\n")
      (org-set-emph-re 'org-emphasis-regexp-components
                       org-emphasis-regexp-components)
      (setq-local org-emphasis-alist '(("*" bold)
                                       ("/" italic)
                                       ("_" underline)
                                       ("=" org-verbatim verbatim)
                                       ("~" org-kbd)
                                       ("+"
                                        (:strike-through t))))
      (setq-local org-hide-emphasis-markers t))
```

@JAremko where did that ^ code go, and would it be appropriate to move into this function?